### ISSUE TITLE
Is this repository broken?

### ISSUE THREAD

**Issue by eladmeir** [zhangxueting/DCN#5]
Hey all
I was completely positive while reading the paper, which seems like a solid and robust idea that can work pretty good in real life applications, until I have tried to use this code

I have managed to make it train (both embeddings and relation, due to lack of pre-trained weights...), after "wild" changed in code. For exmample, I have replaced 
`for image,label in train_dataloader:`
on train.py, line 84, to
`for _ in range(len(train_dataloader.dataset.image_roots)):
            image, label, _ = next(iter(train_dataloader))`

But I have no confidence in such big changes, so I wanted to first make sure that this repo is even "valid" for use.

Has anyone tried and made it work?

Thanks
